google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -113,7 +113,7 @@
       20 * 1024 * 1024; // 20MB API maximum message size.
   private static final int INITIAL_ACK_DEADLINE_SECONDS = 10;
   private static final int MAX_ACK_DEADLINE_SECONDS = 600;
-  private static final int MIN_ACK_DEADLINE_SECONDS = 10;
+  static final int MIN_ACK_DEADLINE_SECONDS = 10;
```

Comment:
It's not just for tests though, it is also used in `PollingSubscriberConnection`.